doc/specs/top_10.md

```diff
@@ -0,0 +1,106 @@
+# Top 10
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+Users are notified of the top 10 most popular albums on the website. Users can view the list in a homepage widget/module. Users can see the most popular albums by past day, past week, past month, and all-time.
```

Comment:
Is this limited to just albums? Also, meteor-gazelle isn't music specific, so refrain from using terms like album, music, etc.